Cargo.toml

```diff
@@ -16,10 +16,9 @@ travis-ci = { repository = "gimli-rs/addr2line" }
 
 [dependencies]
 gimli = { version = "0.20", default-features = false, features = ["read"] }
-fallible-iterator = { version = "0.2", default-features = false }
+fallible-iterator = { version = "0.2", default-features = false, optional = true }
```

Comment:
This is a required dependency of `gimli` with the `read` feature anyway.

Reply:
I, too, am using `cargo tree` and am not simply blindly trying to do things. There's a lot of work that needs to be done [and I'm trying to do it incrementally](https://github.com/gimli-rs/gimli/pull/495).